frontend/public/components/modals/error-modal.tsx

```diff
@@ -31,9 +42,32 @@ export const ModalErrorContent = (props: ErrorModalProps) => {
   );
 };
 
+export const ErrorModal: OverlayComponent<ErrorModalProps> = (props) => {
+  const { t } = useTranslation();
+  const { error, title } = props;
+  const titleText = title || t('public~Error');
+  return (
+    <Modal isOpen onClose={props.closeOverlay} variant={ModalVariant.small}>
+      <ModalHeader title={titleText} titleIconVariant="warning" labelId="title-icon-modal-title" />
+      <PfModalBody>{error}</PfModalBody>
+      <PfModalFooter>
+        <Button key="cancel" variant={ButtonVariant.primary} onClick={props.closeOverlay}>
+          {t('public~OK')}
+        </Button>
+      </PfModalFooter>
+    </Modal>
```

Comment:
when I did similar work with the delete modal, the `Backdrop` component was necessary. Does this work without it? I would expect there to be some style discrepancy.
https://github.com/openshift/console/blob/b33ff3ee10e5dcf076bee83e5dcba1de40900c6e/frontend/public/components/modals/delete-modal.tsx#L163